recipe/meta.yaml

```diff
@@ -17,7 +17,7 @@ source:
   sha256: ba983dbb91a7f51553315fe14f1fbcbb4920f8f2e9bb47620735d2d63c33e673
 
 build:
-  number: 1
+  number: 2
```

Comment:
Since the recipe is not changing, there is no need to be bump the `build/number` here

Reply:
fixed